distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -1,11 +1,11 @@
 
-The Solr Catalog Provider (SCP) is an implementation of the `CatalogProvider` interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
+The Solr Provider is an implementation of the Provider interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
```

Comment:
❓ Check with @ricklarsen to see if this file is still needed. If so, we may need to rework the content as some of it is already covered in other sections.